### PR TITLE
WorkloadInsights onboarding templates info change

### PR DESCRIPTION
**Pull request by heyaWorld**
## PR Checklist

* [x] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.
* [ ] *if updating templates, it is helpful to post a screenshot of what the changes look like.*
* [x] *if updating templates, ensure that your parameters and steps have meaningful names.*
* [ ] *if adding new templates, or changing items in a gallery, it's helpful to post a screenshot of what the gallery looks like now*

### CODE REVIEW

File: Workbooks/Workloads/Onboard MySQL Insights/Onboard MySQL Insights.workbook [hunk 1]
```diff
@@ -156,7 +156,7 @@
...
+                    "json": "MySQL is the world's most popular Open Source SQL database management system. MySQL is highly scalable and can be operated from single desktops to clusters of machines. MySQL also offers hallmark reliability and availability features from master/slave replication configurations and specialized failover clusters. \n\nWith Azure Monitor for MySQL you can manage your MySQL servers with ease. With insights on performance, networking, and InnoDB, Azure Monitor for MySQL summarizes all the important metrics to keep your MySQL server up and running. Make configuration a breeze with templates, automated alerts and customizable actions.\n<br />\n<br />\n[Watch Video]()<br />\n[Learn More]()<br />\n[Documentation]()<br />"
```
Comment (acearun) [inline Workbooks/Workloads/Onboard MySQL Insights/Onboard MySQL Insights.workbook:159]: We should take out the links at the bottom for the private preview or until we have actual working links.

File: Workbooks/Workloads/Onboard Nginx Insights/Onboard Nginx Insights.workbook [hunk 2]
```diff
@@ -156,7 +156,7 @@
...
+                    "json": "NGINX is a high performance open source software designed for a variety of uses ranging from web serving to load balancing. As one of the fastest web servers, NGINX powers the majority of the top websites. NGINX can serve as a replacement for your physical hardware load balancers and improve the performance of your server, while reducing CPU and RAM consumption.\n\nWith Azure Monitor for NGINX, you can manage your NGINX cluster while also gaining visibility into your operating system – Windows or Linux. Make configuration a breeze with templates, automated alerts and customizable actions. Spend less time fighting fires by accurately diagnosing real-time and historical load balancer performance issues, and focus on more proactive measures scale outs.\n<br />\n<br />\n[Watch Video]()<br />\n[Learn More]()<br />\n[Documentation]()<br />"
```
Comment (acearun) [inline Workbooks/Workloads/Onboard Nginx Insights/Onboard Nginx Insights.workbook:159]: Take out links at the bottom

File: Workbooks/Workloads/Onboard Redis Insights/Onboard Redis Insights.workbook [hunk 3]
```diff
@@ -155,7 +155,7 @@
...
+                    "json": "Redis is an open source, in-memory data structure store that has rapidly gained popularity from developers. Redis is known for its fast performance and niche as a system that serves as both a store and cache. Redis functions most commonly as a key-value database, but has also gain prominence as a powerful NoSQL database.\n\nAzure Monitor for Redis combines all the important information you need to manage your Redis servers. Azure Monitor for Redis highlights important performance metrics to ensure your Redis server is running at optimal levels. Make configuration a breeze with templates, automated alerts and customizable actions.\n<br />\n<br />\n[Watch Video]()<br />\n[Learn More]()<br />\n[Documentation]()<br />"
```
Comment (acearun) [inline Workbooks/Workloads/Onboard Redis Insights/Onboard Redis Insights.workbook:158]: Take out links at the bottom

Comment (acearun) [reply Workbooks/Workloads/Onboard Redis Insights/Onboard Redis Insights.workbook:158]: We need content in the pre-requisite tab of MySQL and Redis.

Comment (heyaWorld) [reply Workbooks/Workloads/Onboard Redis Insights/Onboard Redis Insights.workbook:158]: sure

**Comment by acearun** [comment]
We need content in the pre-requisite tab of MySQL and Redis. We should also add instructions on how to construct connection strings in each of the Configure monitoring sections. You can do it as part of a separate PR.

**Comment by acearun** [comment]
Why no content for the SQLs?

**Comment by heyaWorld** [comment]
> Why no content for the SQLs?

Current description for SQL is fine, confirmed with Scott.

**Pull request merged**